### DISCUSSION TITLE
Use Beacon to create, upload, share Store Packages with others

### DISCUSSION THREAD

**Discussion by Beargawn** [gameserverapp/Platform#915] [Suggestions]
Idea:  If Beacon could be used to share the shop packages and upload to the  server owners GameServerApp Web store  as it updates the Ark servers.

**Comment by mvaessen** [comment]
Hi, thanks for your suggestion!

There are no plans to integrate the shop pack system with external applications. It's a system that is developed by GSA and can only be used by GSA.

In the near future the marketplace will be expanded to include shop packs and more.
